### DISCUSSION TITLE
Cannot read property 'url' of null

### DISCUSSION THREAD

**Discussion by fritzgeralds** [jef/streetmerchant#1122] [Help]
I updated to use the rotating proxies but not sure what's happening.
Proxies are working but about 50-60% of the time I get "Cannot read property 'url' of null"

I've created {storename}.proxies in the root for all my stores.
Using http proxies entered as "http://12.34.56.78:3128"
Proxies are from DreamProxies and use IP based authentication. No so no "user:pass@"
Am I doing something wrong?

**Comment by FlyingToto** [comment]
I have the same issue after adding a store.proxies file as well... note that in my case, I am using a single tor proxy localhost:9150... it seems that the error happens less often by cut and pasting the same entry multiple times.... 
socks5://127.0.0.1:9150
socks5://127.0.0.1:9150
socks5://127.0.0.1:9150
socks5://127.0.0.1:9150
socks5://127.0.0.1:9150

My guess is that sometimes tor proxy takes you to faulty/blocked  exit resulting in this error message at which point the program will move to the next in line. immediately.. but not sure...
